app/resources/game_content.py

```diff
@@ -0,0 +1,209 @@
+from vars import hardware, resolve_ram_type, resolve_gpu_type
+from typing import Tuple, List
+import math
+from models.hardware import Hardware
+from models.device import Device
+from models.service import Service
+from models.workload import Workload
+from app import m
+
+
+def check_exists(user: str, elements: dict) -> Tuple[bool, dict]:
+    response: dict = m.contact_microservice("inventory", ["inventory", "list"], {"user": user})
```

Comment:
parameter name has changed to `owner` in cryptic-inventory
```suggestion
    response: dict = m.contact_microservice("inventory", ["inventory", "list"], {"owner": user})
```